website/pages/docs/agent/config-entries/ingress-gateway.mdx

```diff
@@ -343,8 +343,7 @@ Also make two services in the frontend namespace available over a custom port wi
   - `Port` `(int: 0)` - The port that the listener should receive traffic on.
 
   - `Protocol` `(string: "tcp")` - The protocol associated with the listener.
-    This can be any protocol supported by
-    [service-defaults](/docs/agent/config-entries/service-defaults#protocol).
+    Either `tls` or `http`.
```

Comment:
As shown [here](https://github.com/hashicorp/consul/blob/586ee2566f1b75e675adc66a68639ecf22c69ab3/agent/structs/config_entry_gateways.go#L123-L126), `tls` is not a valid input.

```suggestion
    Either `tcp` or `http`.
```

Reply:
Oh man I made the same mistake in the description too. Thanks for catching that!